### DISCUSSION TITLE
macOS arm64 binary release

### DISCUSSION THREAD

**Discussion by jomifepe** [bitwarden/clients#10219] [Password Manager]
### Code Contribution Proposal

Hello, it has been a while since arm64 became the standard for macs, and there still isn't a binary being released for bitwarden. 

Relying solely on npm for the arm64 binary is not a good solution, and with the `macos-14` github runner, it seems like an easy addition, from which many users will benefit from.

Some attempts to release arm64 binaries were already made before [[1]](https://github.com/bitwarden/clients/pull/2976) [[2]](https://github.com/bitwarden/clients/pull/7338), from which I waited a while for some progress, but there hasn't been any in quite some time. 

My proposal is to release a `bw-macos-arm64-*.zip` and `bw-oss-macos-arm64-*.zip` artifacts alongside all the other ones. This can be done using the existing `cli` step in the `build-cli` workflow with some adjustments.

I already made some tests and the release seems to be working, as seen in this successful run: https://github.com/jomifepe/clients/actions/runs/10050097941

Since this only concerns to macOS, and because of the official worker, the contribution will be shorter, with less workarounds, and be easier to review.

I hope this is considered,
Thanks

**Comment by jomifepe** [comment]
Opened a PR for this: https://github.com/bitwarden/clients/pull/10273

**Comment by vgrassia** [comment]
PR https://github.com/bitwarden/clients/pull/10273 has been merged.
